org.adempiere.base/src/org/compiere/model/MAttachment.java

```diff
@@ -58,18 +59,19 @@ public class MAttachment extends X_AD_Attachment
 	/**
 	 * 
 	 */
-	private static final long serialVersionUID = 1447398065894212273L;
+	private static final long serialVersionUID = 5615231734722570658L;
 
 	/**
 	 * 
 	 * @param ctx
 	 * @param AD_Table_ID
 	 * @param Record_ID
 	 * @return attachment or null
+	 * @deprecated Use {@link #get(Properties, int, int, String, String)}

```

Comment:
I found it is kind of counter intuitive that this is mark as deprecated. When I know the target table has _ID key, it seems rather strange that I should supplied the UUID value as well to retrieve the attachment record.